IbisLib/abstractpolydataobject.cpp

```diff
@@ -0,0 +1,515 @@
+/*=========================================================================
+Ibis Neuronav
+Copyright (c) Simon Drouin, Anna Kochanowska, Louis Collins.
+All rights reserved.
+See Copyright.txt or http://ibisneuronav.org/Copyright.html for details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE.  See the above copyright notice for more information.
+=========================================================================*/
+#include <QFileDialog>
```

Comment:
Sorry for being "emmerdeuse" (no English equivalent I know of) I would profit from the occasion to clean the code a little. QFileDialog is not used  neither in AbstractPolyDataObject nor in PolyDataObject. We should not include polydataobject.h here.